src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -60,6 +62,10 @@ export class RiddleOfFire extends BuffWindow {
 			blitzActions: this.blitzActions,
 		}))
 
+		this.addEvaluator(new RiddleOfWindEvaluator({
+			riddleActions: this.riddleActions,
+		}))
```

Comment:
Could this not just use the expected actions evaluator and have an adjust callback to handle expected=0 for the windows we don't get RoW off cooldown? I'm not following why this needs a custom evaluator, especially since it doesn't handle the windows where we know there can't be a RoW.

Reply:
Yeah I think that's a much better option. I've made the change to a expected actions evaluator locally, I'm just curious how we can find out whether or not it's on cool down when in the RoF window. I know `adjustCount` method gives us the events in that window, but I'm not sure if that's helpful for checking whether or not it's on cooldown